11ty/definitions/third-world-country.md

```diff
@@ -0,0 +1,20 @@
+---
+title: third-world country
+slug: third-world-country
+defined: true
+speech: noun
+---
+
+a problematic term used to refer to a country that doesn’t quite align with Western ideology or the Western image of society. (_Also common are terms such as “third-world” (adj.), “first-world”, “developing/developed nations”, etc._)
+
+## History
+
+“Three worlds” terminology originally arose during the cold war to define countries that were aligned with NATO (“1st world”), aligned with the Warsaw Pact (“2nd world”), or non-aligned (“3rd world”). Because many third-world countries were economically poor, it became common to refer to poor countries as third-world.
+
+## Issues
+
+First off, the term is archaic and refers to a /_NATO vs The Communists_/ worldview that is no longer relevant. Secondly, and more importantly, it presupposes a hierarchy between countries, ranked by Western ideals and homogeneity. Western ideology is a fairly narrow way to perceive global socioeconomics. In the words of [Zeeshan Aleem](_https://www.mic.com/articles/107686/why-you-shouldn-t-call-poor-nations-third-world-countries_): “Social democracy in Scandinavia, oil-funded theocracy in Saudi Arabia, and a one-party, partially planned, partially free market economy in China are all vastly different models for generating and harnessing prosperity.”
+
```

Comment:
I think an Impact section here would be helpful.

Some ideas for impact (non-exhaustive):
- Imposes the Western, American/Anglo-centric view of the world
- Misrepresents entire countries and regions as less-than, which dehumanises its people and its progress, reinforcing false stereotypes

Reply:
Absolutely! I'll commit your suggested changes and work on an Impact section.